core/src/main/scala/org/apache/spark/SSLOptions.scala

```diff
@@ -150,8 +150,8 @@ private[spark] object SSLOptions extends Logging {
    * $ - `[ns].enabledAlgorithms` - a comma separated list of ciphers
    *
    * For a list of protocols and ciphers supported by particular Java versions, you may go to
-   * [[https://blogs.oracle.com/java-platform-group/entry/diagnosing_tls_ssl_and_https Oracle
-   * blog page]].
+   * <a href="https://blogs.oracle.com/java-platform-group/entry/diagnosing_tls_ssl_and_https">
+   * Oracle blog page</a>.
```

Comment:
For hyperlinks, it seems

these are fine ([try it](http://htmledit.squarefree.com/)):

```xml
<a href="https://.../...">link ABC</a>
```

```xml
<a href="https://.../...">
link ABC</a>
```

```xml
<a href="https://.../...">link
ABC</a>
```